apps/ltn/tests/ltn_tests.rs

```diff
@@ -0,0 +1,108 @@
+#[cfg(test)]
+mod tests {
+    use tests::{get_test_file_path, import_map, compare_with_goldenfile};
+    use map_model::RoadID;
+    use geom::Pt2D;
+    use ltn::logic::turn_restrictions::FocusedTurns;
+    use ltn::pages::design_ltn::turn_restrictions::handle_edited_turn_restrictions;
+    use ltn::save::Proposals;
+    use abstutil::Timer;
+    use fs_err::File;
+    use std::io::Write;
+
+    #[test]
+    /// Tests that the `get_test_file_path` convenience function itself works as expected.
+    /// Note that this tests is identical to `test_get_test_file_path_tests_crate`. This is
+    /// to assert that the behaviour of `get_test_file_path` is identical from different
+    /// locations within the workspace.
+    fn test_get_test_file_path_ltn_crate() -> Result<(), anyhow::Error> {
```

Comment:
Same comment about not needing `Result` in unit tests

Reply:
Done